addons/stock_account/models/product.py

```diff
@@ -487,6 +487,7 @@ def _svl_empty_stock_am(self, stock_valuation_layers):
                     'credit': abs(value),
                     'product_id': product.id,
                 })],
+                'type': 'entry',
```

Comment:
you don't change the default context, the commit message should be adapted

Reply:
and is this issue present in v13 ?

Reply:
yes its present in V13.0 also. Previously branch was in v13.0 only.

Reply:
can you retarget your branch on v13 ?

Reply:
Branch for v13 is created. PR #43721